examples/otlp/docker/app/main.go

```diff
@@ -0,0 +1,128 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Sample contains a program that exports to the OpenCensus service.
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"math/rand"
+	"os"
+	"time"
+
+	"contrib.go.opencensus.io/exporter/ocagent"
```

Comment:
I am a bit lost on why the demo app from collector is copied here :)

Reply:
yeah...I don't know that we need all the pieces of the demo app. I hadn't trimmed them all out yet. It was a great starting point to have a fully-functional collector back-end with visualizations built in. I'll remove the load generator. :)

Reply:
removed